### PR TITLE
Use destructures

### PR DESCRIPTION
**Pull request by prashant-andani**
making use of destructures for maintainability

### CODE REVIEW

File: packages/events/EventPluginUtils.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ if (__DEV__) {
  * @param {*} inst Internal component instance
  */
 function executeDispatch(event, simulated, listener, inst) {
-  const type = event.type || 'unknown-event';
+  const {type} = event || 'unknown-event';
```
Comment (gaearon) [inline packages/events/EventPluginUtils.js]: This is not equivalent. Instead, it's like `(event || 'unknown-event').type`.

**Comment by gaearon** [comment]
Thanks for PR, but we avoid purely stylistic changes like this. It doesn't increase maintainability in any way. Additionally, this particular change would introduce a bug.

**Comment by prashant-andani** [comment]
Thanks, @gaearon ... It was my first PR though... will look forward to raising a PR that helps in feature development...

**Comment by gaearon** [comment]
No problem, thanks for trying!

**Pull request closed without merging**